### PR TITLE
Adds No nfc-tags text in home

### LINKED ISSUE #397: show information when no tags are in db

### PR DESCRIPTION
**Pull request by schnickalodeon**
closes #397

### CODE REVIEW

File: packages/app/src/views/Home.vue [hunk 1]
```diff
@@ -40,6 +40,20 @@
...
+      <div
+        v-if="tagsOrderedByGroups.length === 0"
+        id="noTags"
+        class="font-primary text-xl text-center my-16 mx-6"
+      >
+        <span class="text-white font-heading text-2xl">
+          <p>
+            Huch, da sind ja keine NFC Tags! <br />
+            Füge fix einen hinzu!
+          </p>
```
Comment (anbraten) [inline packages/app/src/views/Home.vue:52]:
```suggestion
          <p>Huch, da sind ja keine NFC Tags!</p>
          <p>Füge fix einen hinzu!</p>
```

File: packages/app/src/views/Home.vue [hunk 2]
```diff
@@ -40,6 +40,20 @@
...
+          <img src="/src/assets/not-found.gif" />
+        </span>
+        <span></span>
```
Comment (anbraten) [inline packages/app/src/views/Home.vue:55]:
```suggestion
```

File: packages/app/src/views/Home.vue [hunk 3]
```diff
@@ -40,6 +40,20 @@
         :search-input="searchInput"
         @tag-selected="toggleSelectedTag($event)"
       />
+      <div
+        v-if="tagsOrderedByGroups.length === 0"
+        id="noTags"
+        class="font-primary text-xl text-center my-16 mx-6"
+      >
+        <span class="text-white font-heading text-2xl">
```
Comment (anbraten) [inline packages/app/src/views/Home.vue:48]:
```suggestion
        <div class="text-white font-heading text-2xl">
```

File: packages/app/src/views/Home.vue [hunk 4]
```diff
@@ -40,6 +40,20 @@
...
+          <p>
+            Huch, da sind ja keine NFC Tags! <br />
+            Füge fix einen hinzu!
+          </p>
+          <img src="/src/assets/not-found.gif" />
+        </span>
```
Comment (anbraten) [inline packages/app/src/views/Home.vue:54]:
```suggestion
        </div>
```

**Review by anbraten — CHANGES_REQUESTED** [review]

**Review by tcoro — CHANGES_REQUESTED** [review]
I agree with @anbraten

**Review by tcoro — APPROVED** [review]
good

**Review by anbraten — APPROVED** [review]

**Pull request merged**